models.rb

```diff
@@ -106,7 +110,17 @@ def download_scores
       return
     end
 
-    update_scores(updated)
+    rank.nil? ? update_scores(updated) : updated.select { |score| !IGNORED_PLAYERS.include?(score['user_name']) }.uniq { |score| score['user_name'] }[rank]
+  end
+
+  def analyze_replay(replay_id)
```

Comment:
Gotcha. I had a comment initially, but it ended up a bit long and since comments are scarce in inne I assumed you preferred brevity in this aspect. I've now added a succinct but clear one.

Reply:
'The first frame is fictional and must be ignored.' 😂 

Love it. I wish I had comments this informative at work.